things/api/logging.go

```diff
@@ -79,11 +79,11 @@ func (lm *loggingMiddleware) ViewThing(ctx context.Context, token, id string) (t
 	return lm.svc.ViewThing(ctx, token, id)
 }
 
-func (lm *loggingMiddleware) ListThings(ctx context.Context, token string, offset, limit uint64, name string, metadata things.Metadata) (_ things.Page, err error) {
+func (lm *loggingMiddleware) ListThings(ctx context.Context, token string, pm things.PageMetadata) (_ things.Page, err error) {
 	defer func(begin time.Time) {
 		nlog := ""
-		if name != "" {
-			nlog = fmt.Sprintf("with name %s ", name)
+		if pm.Name != "" {
+			nlog = fmt.Sprintf("with name %s ", pm.Name)
```

Comment:
It's not related to this PR, but can you, please, fix the formatting: `nlog = fmt.Sprintf("with name %s", pm.Name)` (no space after) and later `message := fmt.Sprintf("Method list_things %s for token %s took %s to complete", nlog, token, time.Since(begin))` (`%s` surrounded with spaces)? Same for the `ListChannels`.